autonomy/cli/helpers/chain.py

```diff
@@ -61,6 +59,15 @@
 from autonomy.configurations.base import PACKAGE_TYPE_TO_CONFIG_CLASS, Service
 
 
+try:
+    from aea_ledger_ethereum_hwi.exceptions import HWIError
```

Comment:
as next steps: probly apply plugin discovery approach used for aea plugins?
implemented here `https://github.com/valory-xyz/open-aea/blob/main/aea/crypto/plugin.py`
instead of hardcode `try..import`


probably will require plugin updates, so can not be implemented adhoc.